src/components/auth/AuthButton.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Button from "../../views/Button/index";
+import { Logout } from "../../redux/actions/loginAction";
+
+class AuthButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onLogout = this.onLogout.bind(this);
+  }
+
+  onLogout(event) {
+    const { dispatch } = this.props;
+    event.preventDefault();
+    dispatch(Logout());
+  }
+
+  render() {
+    const { isAuthenticated } = this.props;
+
+    return (
+      <div>
+        {isAuthenticated ? (
```

Comment:
This component looks exact to a component I saw on @mikenthiwa's PR??? Anyway, check out the arrangement for a long ternary operator right here https://github.com/airbnb/javascript/issues/1281, the first one with comment //better

Reply:
They are the one handling it. The formatting for this section is being handled by @mikenthiwa and @Bileonaire I was only interested in having my `forgot password` on the login form.